go.mod

```diff
@@ -106,8 +106,9 @@ require (
 	github.com/grafana/grafana v1.9.2-0.20230216173926-a0bea04a0274
 	github.com/julienschmidt/httprouter v1.3.0
 	github.com/kubevela/workflow v0.6.0
-	github.com/oam-dev/kubevela v1.9.1
```

Comment:
kubevela v1.9.5 imports some API break changes. You can rollback to v1.9.4 if you don't want to import too much unrelated changes in this PR.

Reply:
Modified, please help me take a look again